### ISSUE TITLE
[a11y] /careers pages accessibility improvements

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15338]
# Axe-core® Accessibility Results for /en-US/careers/listings/

Page URL: https://www-dev.allizom.org/en-US/careers/listings/

## axe-core found 1 violation

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | Heading levels should only increase by one | heading-order | Best practice | moderate | 1   |

### Failed

1. Heading levels should only increase by one

[Learn more](https://dequeuniversity.com/rules/axe/4.10/heading-order?application=playwright)

heading-order

Best practice

Ensure the order of headings is semantically correct

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>th:nth-child(1) > h4<br>```<br><br>**Element source**<br><br>```<br><h4>Job Title</h4><br>``` | Fix any of the following:<br><br>*   Heading order invalid |

# Axe-core® Accessibility Results for /en-US/careers/ (desktop) project

Page URL: 

- https://www-dev.allizom.org/en-US/careers/
- https://www-dev.allizom.org/en-US/careers/benefits/
- https://www-dev.allizom.org/en-US/careers/diversity/
- https://www-dev.allizom.org/en-US/careers/locations/
- https://www-dev.allizom.org/en-US/careers/teams/

## axe-core found 1 violation

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | All page content should be contained by landmarks | region | Best practice | moderate | 1   |

### Failed

1. All page content should be contained by landmarks

[Learn more](https://dequeuniversity.com/rules/axe/4.10/region?application=playwright)

region

Best practice

Ensure all page content is contained by landmarks

moderate

Issue Tags: cat.keyboard best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.c-careers-bottom<br>```<br><br>**Element source**<br><br>```<br><section class="c-careers-bottom"><br>``` | Fix any of the following:<br><br>*   Some page content is not contained by landmarks |

**Comment by alexgibson** [comment]
Tip: 

- The heading levels in the first violation should be `<h3>` instead of `<h4>`
- The second group of violations can likely all be fixed by making `.c-careers-bottom` an `<aside>` element instead of a `<main>` element.

**Comment by clarapenn** [comment]
@stevejalim can I work on this one, please? Thanks
